app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Filter applications by subject
+description: Let users filter the applications list by the course’s subject
+date: 2021-02-25
+---
+
+We introduced a subject filter to allow users to filter by the course’s subject in the applications list.
+
+## Why we are doing this
+
+Providers need to be able to filter their applications by subject.
+
+When providers share applications with people who do the shortlisting (for example, subject leads in a school or a subject tutor at an HEI), those users only need to see specific subjects and not the entire list.
```

Comment:
We don't allow for sharing. The user is either in the system or their not.

Reply:
Okay, sentence removed.